classes/PodsAPI.php

```diff
@@ -8619,4 +8625,39 @@ public function __call ( $name, $args ) {
         else
             pods_deprecated( "PodsAPI::{$name}", '2.0' );
     }
+
+	/**
+	 * Filter an array of arrays without causing PHP notices/warnings.
+	 *
+	 * @param array $values
+	 *
+	 * @return array
+	 *
+	 * @since 2.6.10
+	 */
+	private function array_filter_walker( $values = array() ) {
+
```

Comment:
That sounds good, but it's not verifying since it only runs on an array where it's called, would be open to adding a check but not sure it's entirely necessary since it's private